launcher/minecraft/auth/AccountData.cpp

```diff
@@ -314,6 +314,8 @@ bool AccountData::resumeStateFromV3(QJsonObject data) {
         type = AccountType::MSA;
     } else if (typeS == "Mojang") {
```

Comment:
I can't see full context but can't this be a switch:case?

Reply:
It could be turned into one, sure. I tried to stay as close as possible to upstream to make merging easier. But if you want I'll change it.

Reply:
Go ahead. We can always resolve upstream merge conflicts

Reply:
I just learned that switch statements only work with integers in C++, so this can't be rewritten.

Reply:
Been doing too much Java recently (Robotics lol). Cool